arangod/Graph/Enumerators/WeightedTwoSidedEnumerator.h

```diff
@@ -101,9 +98,29 @@ class WeightedTwoSidedEnumerator {
 
     [[nodiscard]] bool isEmpty() const { return _queue.empty(); }
 
-    [[nodiscard]] std::vector<CalculatedCandidate> getQueue() const& {
-      return _queue;
-    };
+    [[nodiscard]] std::vector<Step*> getLeftLooseEnds() {
```

Comment:
These are the same changes as the ones in the separate PR https://github.com/arangodb/arangodb/pull/20366
Once the other PR is merged, then the changes will be in devel and this PR will have a smaller change surface.

Reply:
removed that change in favour of #20366